haas/ext/switches/brocade.py

```diff
@@ -139,7 +139,7 @@ def _enable_and_set_mode(self, interface, mode):
         # Enable switching
         url = self._construct_url(interface)
         payload = '<switchport></switchport>'
-        self._make_request('POST', url, data=payload)
+        self._make_request('POST', url, data=payload, log=False)
```

Comment:
Can you check for the expected error code in the case of the switchport already being enabled and log if it's a different one?